smoke-tests/solarwinds-apm-settings.json

```diff
@@ -1,19 +1,19 @@
 [
     {
         "arguments": {
-            "BucketCapacity": 1000000,
+            "BucketCapacity": 800000,
             "BucketRate": 1.0,
             "MetricsFlushInterval": 60,
-            "TriggerRelaxedBucketCapacity": 1000000,
+            "TriggerRelaxedBucketCapacity": 800000,
             "TriggerRelaxedBucketRate": 1.0,
-            "TriggerStrictBucketCapacity": 1000000,
+            "TriggerStrictBucketCapacity": 800000,
             "TriggerStrictBucketRate": 1.0
         },
         "flags": "SAMPLE_START,SAMPLE_THROUGH_ALWAYS,SAMPLE_BUCKET_ENABLED,TRIGGER_TRACE",
         "layer": "",
         "timestamp": 1698176407,
         "ttl": 120,
         "type": 0,
-        "value": 1000000
+        "value": 800000
```

Comment:
i see the `release-test` failed but from quick glance it seemed due to a dependency not found... wondering if going to 80% would make much of a difference to reduce the "compounded-commits" spike in traces, wdyt dialing it further down to 50%?